article/sections/_2_graph_importance.tex

```diff
@@ -0,0 +1,189 @@
+\[ U_{\textrm{sr3+}} = ASC_{\textrm{sr3+}} + \beta_{\textrm{time\_drive}} \times \textrm{Travel\_Time} + \beta_{\textrm{cost\_per\_distance\_sr3+}} \times \textrm{Cost\_per\_Distance}_{\textrm{sr3+}} + \beta_{\textrm{autos}}  \times \textrm{Number\_of\_Autos} + \beta_{\textrm{cross\_bay}} \times \textrm{Cross\_Bay} + \beta_{\textrm{hh\_size}} \times \textrm{HH\_Size} + \beta_{\textrm{n\_kids\_hh}} \times \textrm{Number\_of\_kids} \] \\
+\[ U_{\textrm{WTW}} = ASC_{\textrm{WTW}} + \beta_{\textrm{travel\_time\_transit}} \times \textrm{Travel\_Time} + \beta_{\textrm{travel\_cost}} \times \textrm{Travel\_Cost} \] \\
+\[ U_{\textrm{DTW}} = ASC_{\textrm{DTW}} + \beta_{\textrm{travel\_time\_transit}} \times \textrm{Travel\_Time} + \beta_{\textrm{travel\_cost}} \times \textrm{Travel\_Cost} \] \\
+\[ U_{\textrm{WTD}} = ASC_{\textrm{WTD}} + \beta_{\textrm{travel\_time\_transit}} \times \textrm{Travel\_Time} + \beta_{\textrm{travel\_cost}} \times \textrm{Travel\_Cost} \] \\
+\[ U_{\textrm{Bike}} = ASC_{\textrm{Bike}} + \beta_{\textrm{travel\_distance\_bike}} \times \textrm{Travel\_Distance} \] \\
+\[ U_{\textrm{Walk}} = ASC_{\textrm{Walk}} + \beta_{\textrm{travel\_distance\_walk}} \times \textrm{Travel\_Distance} \] \\
+\end{aligned}
+\end{equation}
+
+For our purposes, we assume that there are no latent variables.
+The variables were chosen based on the specification of the MNL model outlined by the equations above.
+We can describe the variables as follows:
+- Total Travel Distance: is the total travel distance for individual i and mode j, for all available modes for individual i during trip t of tour l.
+- Total Travel Cost: is the travel cost in dollars for individual i and mode j, for all available modes for individual i during trip t of tour l.
+- Total travel time: is the travel time in minutes for individual i and mode j, for all available modes for individual i during trip t tour l.
+- Number of Autos: is the number of automobiles owned by individual i's household.
+- Number of Licensed Drivers: is the number of licensed drives in individual i's household.
+- Number of Kids: is the number of kids in individual i's household.
+- Cross-bay trip: is a binary variable indicating whether the trip t in tour l for individual i is a cross-bay trip.
+
+Figure \ref{fig:IND_GRAPH} illustrates the DAG where all explanatory variables are independent of each other for each utility equation.
+Similarly, Figure \ref{fig:DA_causal_2} through \ref{fig:BIKE_causal_2} illustrate the causal graphs with ``interacting'' explanatory variables.
+Each of these graphs is based on the utility function of each mode in the multinomial logit model specified in \citet{brathwaite_asymmetric}.
```

Comment:
+1 to Hassan.

I'd say ``dependent'' explanatory variables as opposed to interacting, since the explanatory variables are literally dependent on one another, and it directly contrasts the case of the explanatory variables being independent of one another.

Reply:
Does this address your comment @hassan-obeid @timothyb0912 

```suggestion
Figure \ref{fig:IND_GRAPH} illustrates the DAG where all explanatory variables are independent of each other for each utility equation. This DAG represents a scenario where an analyst only modifies variables in their dataset that are directly impacted by the proposed intervention, without considering the different relationships between these variables and other explanatory variables in the dataset.
Similarly, Figure \ref{fig:DA_causal_2} through \ref{fig:BIKE_causal_2} illustrate the causal graphs with ``dependent'' explanatory variables. For our purposes, we assume that these DAGs represent the ``true" data generating process, where several variables affect each each other.
Each of these graphs is based on the utility function of each mode in the multinomial logit model specified in \citet{brathwaite_asymmetric}.
```

Reply:
"each each"

Reply:
See #163